draft-js-inline-toolbar-plugin/src/components/Toolbar/index.js

```diff
@@ -52,16 +39,32 @@ export default class Toolbar extends React.Component {
     // when focusing editor with already present selection
     setTimeout(() => {
       if (!this.toolbar) return;
-      const relativeParent = getRelativeParent(this.toolbar.parentElement);
-      const toolbarHeight = this.toolbar.clientHeight;
-      const relativeRect = (relativeParent || document.body).getBoundingClientRect();
-      const selectionRect = getVisibleSelectionRect(window);
 
+      // The editor root should be two levels above the node from
+      // `getEditorRef`. In case this changes in the future, we
+      // attempt to find the node dynamically by traversing upwards.
+      const editorRef = this.props.store.getItem('getEditorRef')();
+      if (!editorRef) return;
+      let editorRoot = editorRef.refs.editor;
+      while (editorRoot.className.indexOf('DraftEditor-root') === -1) {
```

Comment:
hm not sure about this tbh. Isn't there a way to get a parent that is absolutely positioned? I'd investigate that - because this is brittle